assembl/static2/js/app/components/harvesting/harvestingMenu.jsx

```diff
@@ -27,23 +27,29 @@ type State = {
 };
 
 class HarvestingMenu extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      displayExtractsBox: false
-    };
+  static getDerivedStateFromProps() {
+    const { hash } = window.location;
+    if (hash !== '') {
+      const hashExtractId = hash.split('#')[2];
+      if (hashExtractId) {
+        return { displayExtractsBox: true };
```

Comment:
You want to display extracts box only if hashExtractId is in extracts, not expand all box on all posts.

Reply:
yes good catch!